akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/voice/VoiceApp.scala

```diff
@@ -152,20 +153,22 @@ object VoiceApp extends SystemConfiguration {
               }
             }
           case None =>
-            handleUserJoinedVoiceConfEvtMsg(
-              liveMeeting,
-              outGW,
-              eventBus,
-              liveMeeting.props.voiceProp.voiceConf,
-              cvu.intId,
-              cvu.voiceUserId,
-              cvu.callingWith,
-              cvu.callerIdName,
-              cvu.callerIdNum,
-              cvu.muted,
-              cvu.talking,
-              cvu.calledInto
-            )
+            if (!cvu.intId.startsWith("v_")) {

```

Comment:
Just a thought: there are four occurrences of this check (startsWith("v_") in this PR. I think there might also be a couple of checks in the current trunk head. Maybe it should be centralized into a helper function since it relies on a hardcoded prefix?